src/sentry/notifications/helpers.py

```diff
@@ -715,6 +719,47 @@ def get_recipient_from_team_or_user(user_id: int | None, team_id: int | None) ->
     return recipient
 
 
+def team_is_valid_recipient(team: Team | RpcActor) -> bool:
```

Comment:
I'd add a docstring here just explaining what a "valid" team is. lgtm otherwise!